apps/mitt-konto/src/Components/Paywall.purs

```diff
@@ -23,6 +31,8 @@ import React.Basic.Hooks (Component, component, useEffect, useState, useState',
 import React.Basic.Hooks as React
 import Routing.PushState (PushStateInterface)
 
+type Setter a = a -> Effect Unit
```

Comment:
Just write out the types, it's easier to read than custom wrappers like these, especially for this simple types like these.